docs/docs/user_guide/sidebar/common/studies/introduction.md

```diff
@@ -4,19 +4,17 @@ title: Studies Introduction
 
-Service Workbench also allows an organization to provide access to their data sets, or a subset of their data sets, to external organizations in a controlled way. Further to this, the external organization can use their own AWS account for the research workspace and access the data in the hosting organization.
+Service Workbench also allows an organization to provide access to Studies to external organizations with access control. 
 
-A user can mount zero or more Studies from each of the levels to their [Workspaces](/user_guide/sidebar/common/workspaces/introduction).
-
-There are 3 types of Studies available.
+There are 3 types of Studies available. Users can mount multiple Studies, of any type, to their [Workspaces](/user_guide/sidebar/common/workspaces/introduction).
 
 | Type                     | Description                                                                                                                                                                                                                                                                                               |
 | :----------------------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| **My Studies**           | Studies that are only available to the user that created them. A user can use this to work on datasets that are exclusive to them or that are used specifically for their research. Users can also grant permission to other users in order to allow them access to their studies.                        |
-| **Organization Studies** | Studies that have been shared with the Organization. These could be data that had been collected by efforts of the organization or are licensed to the organization. It is possible to grant or deny users access to this data in order to comply with regulations or licensing restrictions on the data. |
-| **Open Data**            | Service Workbench provides access to AWS Open Data Data Sets by frequently scanning the set of AWS Open Datasets and adding new datasets to this category. This can include the 1000 genomes and other datasets openly available through Amazon.                                                          |
+| **My Studies**           | Studies owned by the user that created them. A user can use this to work on datasets that are exclusive to them or that are used specifically for their research. Users can also grant permission to other users in order to allow them access to their studies.                        |
```

Comment:
"Users can also grant permission to other users in order to allow them access to their studies. " This part isn't accurate about My Studies. They are just private.

Reply:
Fixed in latest commit below.